File: santiago/A.Programacion/Ejercicio6.c
#include<stdio.h>
main()
{	int segundos;
	int dias;
	int horas;
	int minutos;

	segundos=0;
	dias=0;
	horas=0;
	minutos=0;

	printf("Pasar segundos a horas dias y minutos\n");
	printf("Segundos");
	scanf("%d",&segundos);

	dias=segundos/86400;
	horas=segundos/3600;
	minutos=segundos/60;

	printf("Dias:%d\n",dias);
	printf("Horas:%d\n",horas);
	printf("Minutos:%d\n",minutos);
}

File: Tere/python/ejercicio6.py
#6-Escriba un programa que pida una cantidad de segundos
#y que escriba cuántos minutos y segundos son.

print("Escribe unos segundos: ")
segundos= input()

minutos= int(segundos)/60


seg = int(segundos) % 60
print (int(minutos), "minutos y", int(seg), "segundos")


#7-Escriba un programa que pida una cantidad de segundos y
#que escriba cuántas horas, minutos y segundos son.

print("Escribe unos segundos: ")
segundos= input()

horas= int(segundos) /3600

minutos= int(segundos)/60

seg = int(segundos) % 60
print (int (horas),"horas,",int(minutos), "minutos y", int(seg), "segundos")

